ark/phenotyping/pixel_consensus_cluster.R

```diff
@@ -112,23 +128,24 @@ for (batchStart in seq(1, length(fovs), batchSize)) {
                 # this won't be displayed to the user but is used as a helper to break out
                 # in the rare first FOV hang issue
                 print(paste('Done writing fov', fovs[i]))
-                0
+                c(0, '')
             },
             error=function(cond) {
                 # this won't be displayed to the user but is used as a helper to break out
                 # in the rare first FOV hang issue
                 print(paste('Error encountered for fov', fovs[i]))
-                1
+                c(1, cond)
             }
         )
 
-        data.frame(fov=fovs[i], status=status)
+        data.frame(fov=fovs[i], status=status[1], errCond=status[2])
     }
 
     # report any erroneous feather files
     for (i in 1:nrow(fovStatuses)) {
         if (fovStatuses[i, 'status'] == 1) {
-            print(paste("The data for FOV", fovStatuses[i, 'fov'], "has been corrupted, removing"))
+            print(paste("Processing for FOV", fovStatuses[i, 'fov'], "failed, removing from pipeline. Error message:"))
+            print(fovStatuses[i, 'errCond'])
```

Comment:
Were you able to find an easy way to test what happens when too many cores are specified, confirming that this now shows the correct error message?

Reply:
@ngreenwald yes, set the number of cores to an extremely high value (say, 128 on a laptop) and it now displays the error message `"Too many cores specified"`.